docs/admin/third-party.rst

```diff
@@ -0,0 +1,100 @@
+Third-party Components
```

Comment:
You probably must copy this whole file into its new place.

Reply:
Done